data/capture_buffer.go

```diff
@@ -6,6 +6,8 @@ import (
 	v1 "go.viam.com/api/app/datasync/v1"
 )
 
+const captureAllFromCamera = "CaptureAllFromCamera"
```

Comment:
made this a constant as it is now referenced in multiple places.

Reply:
Nit, but is there not some other const somewhere in rdk that defines this method name? I'm surprised we hard-coded it in the first place.

Reply:
It is in RDK but go enforces that packages may not have import cycles and to import this from the vision package, it would create an import cycle.

This will also go away when we complete migrating vision.CaptureAllFromCamera from being stored as tabular data to being stored as binary data.

Reply:
Ah, import cycles.... seems reasonable especially with this going away soon, thanks for explaining.